### PR TITLE
`FeatureFormView` / `ComboBoxInput` - Show unsupported values

### PR DESCRIPTION
**Pull request by dfeinzimer**
Apollo 839

https://github.com/user-attachments/assets/4ed991a6-e349-4c54-8a0b-96118b10880f

### CODE REVIEW

**Comment by dfeinzimer** [comment]
Note that in its current form, this doesn't show the current unsupported type in the options list like Kotlin. Thoughts on this?

<p align="center">
  
</p>

**Comment by dfeinzimer** [comment]
Used map `99ef6889b8ab42eb8cbd3b9672c7b592` / "Extrinsic_Value" to test fallback behavior successfully.

<p align="center">
  
</p>

**Comment by mhdostal** [comment]
> Note that in its current form, this doesn't show the current "Unsupported Type" in the options like Kotlin. Thoughts on this?

Is there a reason we're not following what Kotlin does?

File: Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/ComboBoxInput.swift [hunk 1]
```diff
@@ -25,9 +25,6 @@ struct ComboBoxInput: View {
     /// The phrase to use when filtering by coded value name.
     @State private var filterPhrase = ""
     
-    /// The formatted version of the element's current value.
-    @State private var formattedValue = ""
-
```
Comment (dfeinzimer) [inline Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/ComboBoxInput.swift:30]: `formattedValue` was unused

**Comment by dfeinzimer** [comment]
> Is there a reason we're not following what Kotlin does?

My original thinking was that it was overcomplicating the sheet content but I can see now how it helps to provide a bit of context around the value being invalid. 

I've moved to match Kotlin.



https://github.com/user-attachments/assets/e86704b0-cbc1-4331-b2dd-c91734858dca


https://github.com/user-attachments/assets/0bbba34e-6839-4c2c-9fff-fcc6244bd1e3

**Review by mhdostal — APPROVED** [review]

**Comment by dfeinzimer** [comment]
Verified all Feature Form tests pass on iOS 17 and 18 simulators

**Pull request merged**